lib/reconcile/rabbitmq.go

```diff
@@ -106,6 +106,11 @@ func EnsureRabbitMQ(cr *apiv1alpha2.Astarte, c client.Client, scheme *runtime.Sc
 		return nil
 	}
 
+	// Add deprication notice for rabbitmq deployed by astarte operator
+	if pointy.BoolValue(cr.Spec.RabbitMQ.Deploy, true) {
```

Comment:
This `if` statement can be removed. In case RabbitMQ is not deployed we return, and this line is not executed.